kubeflow_batch_predict/dataflow/batch_prediction.py

```diff
@@ -226,8 +232,8 @@ def process(self, element, model_dir):
           td.microseconds / 10**3  + (td.seconds + td.days * 24 * 3600) * 10**3)
       self._batch_process_ms_distribution.update(time_delta_in_ms)
 
-      for i, p in zip(inputs, predictions):
-        yield i, p
+      loaded_data[self._predictions_key] = predictions
```

Comment:
If you do the suggestion in line 214, you need to do a "zip" to match the predictions with other "passthrough" fields.